internal/config/cgmgr/cgmgr_linux.go

```diff
@@ -59,13 +59,29 @@ type CgroupManager interface {
 	// PopulateContainerCgroupStats fills the stats object with information from the cgroup found
 	// given a cgroup parent and container ID.
 	PopulateContainerCgroupStats(sbParent, containerID string, stats *types.ContainerStats) error
+	// ContainerCgroupStats returns a stats object with information from the cgroup found
+	// given a cgroup parent and container ID.
+	ContainerCgroupStats(sbParent, containerID string) (*CgroupStats, error)
```

Comment:
is this called anywhere? I think I'd prefer to not have this endpoint for now. The caller of ContainerCgroupManager can implement it, no?

Reply:
it isn't called anywhere yet, not in this PR. Later I am planning of removing `PopulateContainerCgroupStats` and similar functions, and use `ContainerCgroupStats` and `SandboxCgroupStats`. Both  `ContainerCgroupStats` and `SandboxCgroupStats` should be called directly from the stats server's loop.